examples/call_multiple_tools_with_llm.py

```diff
@@ -0,0 +1,84 @@
+"""
```

Comment:
Same example as `call_multiple_tools_directly_with_auth.py` but instead it uses an LLM